src/main/java/bio/terra/controller/RepositoryApiController.java

```diff
@@ -132,11 +140,36 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
         return new ResponseEntity<>(studyService.delete(UUID.fromString(id), getAuthenticatedInfo()), HttpStatus.OK);
     }
 
-    public ResponseEntity<List<StudySummaryModel>> enumerateStudies(
-            @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
-            @RequestParam(value = "limit", required = false, defaultValue = "10") Integer limit) {
-        validiateOffsetAndLimit(offset, limit);
-        return new ResponseEntity<>(studyService.enumerate(offset, limit), HttpStatus.OK);
+    private void validateEnumerateParams(Integer offset, Integer limit, String sort, String direction) {
+        List<String> errors = new ArrayList<>();
+        if (offset < 0) {
+            errors.add("offset must be greater than or equal to 0.");
+        }
+        if (limit < 1) {
+            errors.add("limit must be greater than or equal to 1.");
+        }
+        if (sort != null && sort.length() > 0 && !VALID_SORT_OPTIONS.contains(sort)) {
```

Comment:
you can use this to check that sort is not null and has a value
!StringUtils.isEmpty(sort)

same with direction below